package.json

```diff
@@ -49,7 +49,8 @@
     "urkel": "~0.6.3"
   },
   "devDependencies": {
-    "bmocha": "^2.1.5"
+    "bmocha": "^2.1.5",
+    "jsdoc": "^3.6.6"
```

Comment:
I'm not opposed to including this check in CI, but can we do it without adding a dependency? 

Notice how github actions has to install bslint on its own when it installs hsd. Contributors could be advised to install jsdocs in https://github.com/handshake-org/hsd/blob/master/CONTRIBUTING.md#code-style the same way they are advised to install eslint.